strawberryfields/decompositions.py

```diff
@@ -163,10 +164,19 @@ def nullT(n, m, U):
 
     return [n-1, n, thetar, phir, nmax]
 
-def clements(V, tol=1e-11):
-    r"""Performs the Clements decomposition of a Unitary complex matrix.
+def clements(V, tol=11):
+    r"""Clements decomposition with phases in the middle of interferomter
```

Comment:
```suggestion
    r"""Performs the Clements decomposition of a Unitary complex matrix, with local
    phase shifts applied in the middle of the interferometer.
```

Reply:
I like the fact that you labelled the transformations according to the  equation numbering of Clements paper. I am assuming there is no difference between the Optica and arxiv.